<launch>
    <group ns="vision">
			<node name="skinDetectNode" type="skinDetectNode" pkg="vision" respawn="False" output="screen" />
			<!-- Add in upper line to enable debugging: launch-prefix="kdbg" /-->
			
			<!-- Package Path /-->
			<param name="/vision/packagepath" value="$(find vision)" />
			
			<include file="$(find vision)/launch/general_params.launch" />	
			
			<!-- **************** /-->
			<!-- Debug Mode params /-->
			<!-- **************** /-->
				
				<!-- Enable Skin Debug /-->
				<param name="debugSkin" type="bool" value="True"/>
        
			<!-- **************** /-->
			<!-- Dummy Msg params /-->
			<!-- **************** /-->
			
				<!-- Enable Skin Dummy /-->
				<param name="skinDummy" type="bool" value="False"/>
				
            
            <!-- ************** /-->
			<!--  Timer params  /-->
			<!-- ************** /-->
            
				<!-- Timer Period for skin detection  /-->
				<param name="skinTime" type="double" value="0.05"/>
				
			<!-- *********************** /-->
			<!--  SkinDetector params  /-->
			<!-- *********************** /-->
			
				<!-- Filepath for skin (positive) histogramm  /-->
				<param name="skinHist" type="str" value="$(find vision)/data/histogramms/histogramm_skin.jpg"/>
				
				<!-- Filepath for wall (negative) histogramm  /-->
				<param name="wallHist" type="str" value="$(find vision)/data/histogramms/histogramm_wall.jpg"/>
				
				<!-- Filepath for wall ( 2nd negative) histogramm  /-->
				<param name="wall2Hist" type="str" value="$(find vision)/data/histogramms/histogramm_wall2.jpg"/>
				
				<!-- threshold for final size of skin contours  /-->
				<param name="sizeThreshold" type="int" value="300"/>
	</group>
</launch>
